docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -123,3 +123,138 @@ and secret keys, STS, or an IAM role:
 * - `s3.external-id`
   - External ID for the IAM role trust policy when connecting to S3.
 :::
+
+## Security mapping
+
+Trino supports flexible security mapping for S3, allowing for separate
+credentials or IAM roles for specific users or buckets/paths. The IAM role
```

Comment:
can we somehow avoid  "bucket/paths". Maybe even by using "users or buckets or paths" .. even though they are equivalent. What do we do in the rest of the docs or what is happening in the S3 docs for this?

Reply:
I'll say "S3 locations" since it is bucket plus path within the bucket

Reply:
that works for me